resources/building-mobile-extensions/building-an-extension.md

```diff
@@ -31,8 +31,12 @@ To create a simple extension, complete the following procedures in the order in
 
 ### A. **Create an extension class**
 
+
 The `ACPExtension`\(iOS\) or `Extension` \(Android\) class is the base class from which extensions must be derived. The `init` method \(iOS\) or the base `constructor` \(Android\) of your extension class is where you can extend the Adobe Experience Platform SDKs functionality by registering event listeners or by setting a default shared state that other modules can access.
 
+In the Adobe Experience Cloud Platform SDK, the strings  like extension name, event type and source strings are converted and stored in lowercase. When you compare strings use java.lang.String.equalsIgnoreCase() method.
```

Comment:
Let's not add this information here, but add it in the getName API description below. For example:
{% hint style="warning" %}
The name that you use to register **cannot** conflict with other registered extensions or Adobe internal modules. The extension name is considered case insensitive by the Mobile SDK.